### PR TITLE
EVG-8146 Add displayname field to patch build variant

### PR DESCRIPTION
**Pull request by khelif96**
Still need to update the test data but this is proving to be a tedious process.

### CODE REVIEW

File: graphql/resolvers.go [hunk 1]
```diff
@@ -770,20 +770,27 @@ func (r *queryResolver) PatchBuildVariants(ctx context.Context, patchID string)
...
+	variantDisplayName := make(map[string]string)
 	for _, task := range tasks {
 		t := PatchBuildVariantTask{
 			ID:     task.Id,
 			Name:   task.DisplayName,
-			Status: task.GetDisplayStatus(),
+			Status: task.Status,
 		}
 		tasksByVariant[task.BuildVariant] = append(tasksByVariant[task.BuildVariant], &t)
+		build, err := r.sc.FindBuildById(task.BuildId)
```
Comment (john-m-liu) [inline graphql/resolvers.go:781]: Check if the variant was already looked up before doing this query

File: graphql/resolvers.go [hunk 2]
```diff
@@ -770,20 +770,30 @@ func (r *queryResolver) PatchBuildVariants(ctx context.Context, patchID string)
 	if err != nil {
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting tasks for patch `%s`: %s", patchID, err))
 	}
+	variantDisplayName := make(map[string]string)
 	for _, task := range tasks {
 		t := PatchBuildVariantTask{
 			ID:     task.Id,
 			Name:   task.DisplayName,
-			Status: task.GetDisplayStatus(),
+			Status: task.Status,
```
Comment (SupaJoon) [inline graphql/resolvers.go:778]: why do we use `task.Status` instead of `task.GetDisplayStatus()`?

Comment (khelif96) [reply graphql/resolvers.go]: Great Catch! I must have accidentally changed it while trying to figure this out.

**Comment by khelif96** [comment]
Failed tests are unrelated  flakey tests

**Pull request merged**